blocks/core/ff_module/ff_module-task-event/src/TaskEventContainer.rt

```diff
@@ -0,0 +1,6 @@
+<div
+    class="ff_module-task-event ff_module-task-event--{this.props.event.type}"
+    >
+    <time class="ff_module-task-event__sent">{this.props.event.sent}</time>
```

Comment:
Alright, is that something that already lives in Firefly? 
If not, I'm guessing it's something that could end up being widely used... Dd you foresee a problem with it living in the patternlib?